srfi-64.html

```diff
@@ -51,13 +51,13 @@ <h2>Rationale</h2>
 possible to write a portable basic implementation.</p>
 <p>
 There are other testing frameworks written in Scheme, including
-<a href="http://schematics.sourceforge.net/schemeunit.html">SchemeUnit</a>.
-However SchemeUnit is not portable.
+<a href="https://docs.racket-lang.org/rackunit/">RackUnit</a>.
+However RackUnit is not portable.
 It is also a bit on the verbose side.
-It would be useful to have a bridge between this framework and SchemeUnit
-so SchemeUnit tests could run under this framework and vice versa.
+It would be useful to have a bridge between this framework and RackUnit
```

Comment:
Is RackUnit just a new name for SchemeUnit, or is it entirely separate?  If the latter, we can't change that kind of thing in the SRFI since it has been finalized.  I could add a post-finalization note in the Status section, though.  Just let me know.